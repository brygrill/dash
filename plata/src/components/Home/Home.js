import React from 'react';
import { Header } from 'semantic-ui-react';

const Home = () => {
  return (
    <Header
      as="h1"
      textAlign="center"
      content="Home Page"
    />
  );
};

export default Home;
