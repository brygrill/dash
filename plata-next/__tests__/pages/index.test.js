import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../../pages';

test('Home pages shows the home page!', () => {
  const component = renderer.create(<Home />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
