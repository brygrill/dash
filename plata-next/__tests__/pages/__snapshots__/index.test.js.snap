exports[`test Home pages shows the home page! 1`] = `
<div>
  <div />
  <div
    className="ui fluid container">
    <div>
      <div>
        <h1>
          Home Page
        </h1>
      </div>
      <div>
        <a
          href="/login"
          onClick={[Function]}>
          Login
        </a>
      </div>
      <div>
        <a
          href="/dashboard"
          onClick={[Function]}>
          Dashboard
        </a>
      </div>
    </div>
  </div>
</div>
`;
